scripts/validation/providers/validate.ts

```diff
@@ -41,6 +41,13 @@ if (validator.errors) {
     process.exit(1);
 }
 
+const invalidInterpolation = /(?<!(\$|]))\{/g;
```

Comment:
why the `]` condition?

Reply:
there are some regex like [a-z]{10} so it should not warm if match a ]{